include/magma/vk/textures/CustomImageView.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file CustomImageView.h
+ * @brief Class contain custom image view
+ * @version 0.1
+ * @date 2021-02-13
+ */
+#pragma once
+#include <vulkan/vulkan.h>
+
+#include "magma/vk/textures/ImageView.h"
+#include "magma/vk/textures/Texture.h"
+
+class CustomImageView final : public ImageView {
+public:
+    CustomImageView(VkDevice device, Texture &texture, VkImageAspectFlags aspectMask);
```

Comment:
Argument VkDevice device is redundant. You can extract device from texture

Reply:
Done